ui/src/main/resources/index.html

```diff
@@ -52,5 +52,6 @@
       <!-- }}} -->
     </div>
     <script type="text/javascript" src="bundle.js"></script>
+    <script type="text/javascript" src="/admin/iframe-resizer/2.6.2/js/iframeResizer.contentWindow.min.js"></script>
```

Comment:
Seems like that should be safe enough in this case.  We do not have a good way to track who is sharing what between web apps in the admin UI currently.